src/retyped/jbuild

```diff
@@ -2,4 +2,4 @@
 
 (library
   ((name retyped)
-    (libraries (sedlex flow_parser))))
+    (libraries (js_of_ocaml sedlex flow_parser))))
```

Comment:
Unfortunately we can't compile to native if `src/retyped` depends on jsoo 😕

<img width="885" alt="screen shot 2017-07-16 at 2 13 19 pm" src="https://user-images.githubusercontent.com/6325382/28251391-18d54c28-6a31-11e7-9fe1-209d4c6dd632.png">

Is there some other Regex library or implementation we could use?